product/toolbars/containers_center_tb.yaml

```diff
@@ -52,6 +52,12 @@
     :title: Monitoring
     :text: Monitoring
     :items:
+    - :button: container_timeline
+      :image: timeline
+      :text: "Timelines"
+      :title: "Show Timelines for this Container"
+      :url: '/explorer'

```

Comment:
@dkorn i think we don't need to set url here, as we are going thru x_button method